src/core/public/notifications/toasts/toasts_service.mock.ts

```diff
@@ -21,7 +21,7 @@ import { Observable } from 'rxjs';
 import { ToastsSetup } from './toasts_service';
 
 const createToastsApiMock = () => {
-  const api: jest.Mocked<PublicMethodsOf<ToastsSetup>> = {
+  const api: jest.Mocked<ToastsSetup> = {
```

Comment:
`jest.Mocked<IToasts>`? This is the same, but references the original interface

Reply:
I will address this in the original PR: #47069